public/app/plugins/datasource/loki/components/monaco-query-field/monaco-completion-provider/situation.ts

```diff
@@ -116,7 +119,12 @@ export type Situation =
   | {
       type: 'AFTER_SELECTOR';
       afterPipe: boolean;
-      labels: Label[];
+      parser?: string;
```

Comment:
Do we still need this parser here? I thought that we had it here to not offer 2 parsers, but we learned that there might be many parsers. So do we still need it?

Reply:
No, thanks for noticing.